_includes/footer.html

```diff
@@ -9,29 +9,29 @@ <h3>{% t navigation.resources %}</h3>
                                 <li><a class="white" href="{{ site.baseurl }}/resources/developer-guides/">{% t titles.developerguides %}</a></li>
                                 <li><a class="white" href="{{ site.baseurl }}/resources/user-guides/">{% t titles.userguides %}</a></li>
                                 <li><a class="white" href="{{ site.baseurl }}/library/">{% t titles.library %}</a></li>
-                                <li><a class="white" href="{{ site.baseurl_root }}/feed.xml">{% t footer.feed %}</a></li>
+                                <li><a class="white" href="{{ site.baseurl }}/downloads/">{% t titles.downloads %}</a></li>
+                                
                             </ul>
                      </div>
                      <div class="col-sm-3 col-xs-6">
-                        <h3>{% t titles.channels %}</h3>
+                        <h3>{% t footer.reachout %}</h3>
                             <ul class="list-unstyled">
-                                 <li><a class="white" href="irc://chat.freenode.net/#monero">monero</a></li>
-                                 <li><a class="white" href="irc://chat.freenode.net/#monero-dev">monero-dev</a></li>
-                                 <li><a class="white" href="irc://chat.freenode.net/#monero-markets">monero-markets</a></li>
-                                 <li><a class="white" href="irc://chat.freenode.net/#monero-pools">monero-pools</a></li>
-                                 <li><a class="white" href="irc://chat.freenode.net/#monero-community">monero-community</a></li>
-                                 <li><a class="white" href="irc://chat.freenode.net/#monero-translations">monero-translations</a></li>
-                                 <li><a class="white" href="irc://chat.freenode.net/#monero-hardware">monero-hardware</a></li>
+                                 <li><a class="white" href="{{ site.baseurl}}/community/team/">{% t titles.workgroups %}</a></li>
+                                 <li><a class="white" href="{{ site.baseurl}}/community/hangouts/">{% t titles.hangouts %}</a></li>
+                                 <li><a class="white" href="https://lists.getmonero.org/postorius/lists/monero-announce.lists.getmonero.org/">{% t footer.ml %}</a></li>
+                                 <li><a class="white" href="{{ site.baseurl_root }}/feed.xml">{% t footer.feed %}</a></li>
```

Comment:
Would this be better by the legal/source code/sitemap? Seems a little out of place under "Reach Out" to me

Reply:
Agreed.